docs/products/mysql/howto/identify-disk-usage-issues.rst

```diff
@@ -0,0 +1,37 @@
+Identify disk usage issues
+==========================
+
+Aiven for MySQL is configured to use ``innodb_file_per_table=ON`` which means that for every table its data and indexes are stored together in a separate ``.idb`` file.
+
+Over time, when a table receives a lot of inserts and deletions, the amount of space it occupies on disk can grow significantly larger than the current data in the table. A classic example of this would be a table containing jobs for a work queue in which rows are repeatedly added to the end of the table and removed from the beginning.
+
+InnoDB does not release this allocated space back to the operating system automatically, in case the table grows larger in the future, but this can cause problems. Since every other table exists in its own ``.idb`` file, the allocated but unused space is unavailable for the tables to grow.
```

Comment:
I would link this paragraph with the one above
```suggestion
This is due to InnoDB not releasing the allocated space back to the operating system automatically, in case the table grows larger in the future, but this can cause problems. Since every other table exists in its own ``.idb`` file, the allocated but unused space is unavailable for the tables to grow.
```